packages/repository/src/utils/utils.ts

```diff
@@ -0,0 +1,60 @@
+import type { DataRecordValue, LyricRepositoryConfig, SongRepositoryConfig } from '@overture-stack/maestro-common';
+
+/**
+ * Finds the repository by its repository code
+ * @param repositories
+ * @param repoCode
+ * @returns
+ */
+export const getRepoInformation = (
```

Comment:
Description should clearly state that if the repository cannot be found this will return `undefined`